jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/internal/core/ls/JakartaDelegateCommandHandlerForJava.java

```diff
@@ -52,8 +67,14 @@ public Object executeCommand(String commandId, List<Object> arguments, IProgress
 		JavaLanguageServerPlugin
 				.logInfo(String.format("Executing command '%s' in LSP4Jakarta JDT LS extension", commandId));
 		switch (commandId) {
+			case JAVA_CODEACTION_COMMAND_ID:
+                return getCodeActionForJava(arguments, commandId, monitor);
+			case JAVA_CODEACTION_RESOLVE_COMMAND_ID:
```

Comment:
This is supposed to call: resolveCodeActionForJava not etCodeActionForJava

Reply:
ok, thanks for catching that